projects/storefrontlib/src/shared/components/date-time-picker/date-time-picker.component.ts

```diff
@@ -39,11 +42,19 @@ export class DateTimePickerComponent
   nativeValue: string = null;
 
   /**
-   * Reference to input element of type 'datetime-local'.
+   * Reference to input element.
    */
   @ViewChild('inputElement', { static: false, read: ElementRef })
   input: ElementRef;
 
+  /**
+   * Type attribute on input element to manipulate datetime-like value.
+   * Defaults to 'datetime-local'.
+   */
+  @Input()
+  inputType?: DATETIME_PICKER_INPUT_TYPE =
+    DATETIME_PICKER_INPUT_TYPE.DATETIME_LOCAL;
```

Comment:
Yup that's correct for the change to 'date' whilst still having the ability to use the original 'datetime-local' type. I believe that keeping all these date/time controls in one component makes sense in that they all accomplish a similar task and use similar parameters (eg. min/max). 

A fallback mechanism would be ideal but given that this will take more time to implement, we should just use the 'date' type with the 'datetime-local' type there as an option and in the future, we can use the better 'datetime-local' by default with the fallback depending on system/browser support.

We should drop the date-picker component completely as we are not using it anywhere. I'm not sure if we even fully tested it in the end. Its obsolete to the date-time-picker.

Given restrictions of the backend (timezone info missing) and minimum requirements outlined by Bill in the issue, we only need to use date in the budget. Having also time was a workaround for not being able to have the timezone but we agreed just to have date in the ui instead and the time offset would be set by the user's local midnight (more user friendly approach as time is not that important).

Reply:
As Bill agreed to use only date for the budget, I'm in favour of using simple `<cx-date-picker>` component for it (which inside uses native `<input type="date">`.

As `datetime-local` is not supported by all evergreen browsers, IMO we should drop usage of it at all (unless we provide automatic fallback). It's My bad, I didn't notice that a couple of weeks ago when I reviewed the initial datetime picker implementation...

As `datetime-local` is not supported by Firefox, the current implementation of `<cx-date-time-picker>` is not production ready (Spartacus should support all evergreen browsers by default). So I'd suggest hiding this component for now from the public API before 3.0 (and his counterpart formatter service) and add code comment that we should not use this component yet until ticket XXX is done. Then let's create ticket XXX for implementing a stable (working on all browsers) date-time-picker that we could use in other features in future (when needed). That's more complex topic, and not urgent, so ticket implementation can be postponed to 3.x. 

Side note: The occ Scheduled replenishment serializer already uses the `DateTimePickerFormatterService`, but as discussed offline with @bgambocjaviniar , we should drop dependency on the formatter service in the serializer, and rather use only `getLocalTimezoneOffset()` util function.